site/content/en/docs/tasks/run_plain_pods.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Run A Plain Pod"
+date: 2023-09-27
+weight: 6
+description: >
+  Run a Kueue scheduled Pod.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running plain Pods.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. By default, the integration for `v1/pod` is not enabled.
+   Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version)
+   and enable the `pod` integration.
+
+   Example `integrations` section of manager configuration with enabled pod integration:
+   ```yaml
```

Comment:
Add the top level fields, like kind

Reply:
Should I create a minimal working configuration, or just add some of the top level fields?

Should we do the same thing for the following pages?
[run_mpijobs.md](https://github.com/kubernetes-sigs/kueue/blob/main/site/content/en/docs/tasks/run_kubeflow_jobs/run_mpijobs.md?plain=1#L22)
[run_jobsets.md](https://github.com/kubernetes-sigs/kueue/blob/main/site/content/en/docs/tasks/run_jobsets.md?plain=1#L19)

Reply:
just the top level fields, to have a good context of where in the configuration `integrations` fit.

Yes, we should probably do the same for others. You can do that in a follow up.

Reply:
Updated example configuration in adfe10d2